### PR TITLE
Voting: launch fixes

### PR DESCRIPTION
**Pull request by sohkai**
Fixes for launch:

* Removes dummy data
* Fixes NaNs from showing up
* Asks contracts about user balances and if they can vote
* Use rinkeby etherscan URL

### CODE REVIEW

File: apps/voting/app/src/components/VotePanelContent.js [hunk 1]
```diff
@@ -1,37 +1,63 @@
...
+    app: PropTypes.object.isRequired,
+  }
+  state = {
+    userBalance: null,
+    voterState: null,
+  }
+  componentDidMount() {
+    const { app, user, vote } = this.props
+    app
+      .call('getVoterState', vote.id, user)
```
Comment (sohkai) [inline apps/voting/app/src/components/VotePanelContent.js]: @onbjerg This doesn't seem to be working :(

Comment (onbjerg) [reply apps/voting/app/src/components/VotePanelContent.js]: Just leaving this here for note: we should check if `vote.id` even exists (it's not set in `marshallVote`)

**Comment by onbjerg** [comment]
Can you also update to `@aragon/client` beta.6? It includes a new method we need to use for voting descriptions.

Few changes:

- If `vote.description` does not exist, the description section should be hidden.
- If vote has a script attached to it, set `vote.description` to the value returned by `app.describeScript(vote.script)`

`describeScript` takes in a callscript and returns an observable that emits a single array of objects, where each object is in the following shape:

```js
{
  to: string,
  data: string,
  ?description: string
}
```

The vote description should reflect the description of each item in this array. Note that some items may lack a description, if there is no Radspec string for the method being called on that particular address.

**Comment by sohkai** [comment]
@onbjerg will be done in another PR.

**Pull request merged**